2024-10-09 15:21:17,881 Evaluation process is started.
python "C:\Users\DS1DPC2003M\Desktop\tool\neural_network_console3.10\libs\Python\Lib\site-packages\nnabla\utils\cli\cli.py" forward
	-c "C:\Users\DS1DPC2003M\Desktop\spresense_cv\data_set\foopinmodel.files\20241009_144856\results.nnp"
	-d "C:\Users\DS1DPC2003M\Desktop\spresense_cv\dataset2\test.csv"
	-o "C:\Users\DS1DPC2003M\Desktop\spresense_cv\data_set\foopinmodel.files\20241009_144856"
	--replace_path
2024-10-09 15:21:19,094 [nnabla]: [CALLBACK]: Exec forward on local
2024-10-09 15:21:19,112 [nnabla]: Using context "Context(backend=['cpu:float'], array_class='CpuCachedArray', device_id='')"
2024-10-09 15:21:19,154 [nnabla]: data 16 / 50
2024-10-09 15:21:19,159 [nnabla]: data 32 / 50
2024-10-09 15:21:19,166 [nnabla]: data 48 / 50
2024-10-09 15:21:19,173 [nnabla]: data 50 / 50
2024-10-09 15:21:19,176 [nnabla]: Forward Completed.
NNabla command line interface (Version:1.39.0.dev1, Build:231224031435, Callback:NNabla SSH callback module.)
